README.md

```diff
@@ -1979,15 +1979,19 @@ While this guide explains the *what*, *why* and *how*, I find it helpful to see
     function AvengersController() { }
     ```
 
-### Factory Names
+### Factory and Service Names
 ###### [Style [Y125](#style-y125)]
 
-  - Use consistent names for all factories named after their feature. Use camel-casing for services and factories. Avoid prefixing factories and services with `$`.
+  - Use consistent names for all factories and services named after their feature. Use camel-casing for services and factories. Avoid prefixing factories and services with `$`. Only suffix service and factories with `Service` when it is not clear what they are (i.e. when they are nouns).
 
```

Comment:
I wonder if this recommendation will be confusing to those who are not native English speakers